### PR TITLE
add cloud.gov page

### PR DESCRIPTION
**Pull request by wslack**

### CODE REVIEW

File: pages/how-federalist-works/cloud-gov.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+title: Federalist and cloud.gov
+permalink: /pages/how-federalist-works/cloud-gov/
+---
+
+# Federalist and cloud.gov
+
+How they use the other, how much they appreciate the other, why you should look at the other.
```
Comment (jseppi) [inline pages/how-federalist-works/cloud-gov.md]: This seems a little out of place - like maybe it was a writing prompt

File: pages/how-federalist-works/cloud-gov.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+As mentioned elsewhere, Federalist is build on top of the cloud.gov platform-as-a-service (PaaS). cloud.gov takes care of the vast majority of the systems functionality that enables Federalist, especially by maintaining a FedRAMP certification. cloud.gov also provides security, logging, authentication, cryptographic protection, monitoring, and provisioning services that Federalist consumes. Federalist's [compliance memo]({{site.baseurl}}/assets/documents/Federalist-Compliance-Memo.pdf) explains the technical relationship in detail.
```
Comment (jseppi) [inline pages/how-federalist-works/cloud-gov.md]: perhaps link this first instance of cloud.gov to https://cloud.gov

File: pages/how-federalist-works/cloud-gov.md (hunk 1 above)
Comment (wslack) [reply pages/how-federalist-works/cloud-gov.md]: you are right!

File: pages/how-federalist-works/cloud-gov.md (hunk 2 above)
Comment (wslack) [reply pages/how-federalist-works/cloud-gov.md]: done

**Pull request merged**